indico/modules/events/templates/forms/event_person_link_widget.html

```diff
@@ -0,0 +1,10 @@
+{% extends 'forms/_person_link_widget_base.html' %}
+
+{% block sections %}
+    {% if field.get_form().event_type in ('lecture', 'simple_event')  %}
```

Comment:
when do you get `simple_event` here? here you only check for `lecture`: https://github.com/indico/indico/pull/2294/files#diff-aa6104a608c096f1f8305b96b0194e81R90

I'd make sure it always gets the "new" event type (i.e. lecture and not simple_event for a lecture) from the `Event.type` property.